app/Helpers/StatsHelper.php

```diff
@@ -56,6 +56,16 @@ public function getDayStats() {
         return $stats;
     }
 
+    public function getDayStatspgSQL() {
+      $stats = $this->getBaseRows()
+          ->select(DB::raw("to_char(created_at, 'yyyy-mm-dd') AS x, count(*) AS y"))
```

Comment:
Is there any way to perform this action on Postgres and MySQL with the same query? Does Eloquent ORM have any functionality that would allow for this SELECT to be performed without requiring two different functions to be created?

Reply:
Hi,
What if there was a single function as before, and PgSQL and MySQL specific code are implemented in this function? In this case, the controller is not aware of the details of the implementation.

Reply:
@cydrobolt I searched a lot, and can't find any Eloquent function to get a specific date format in a query.

@feuzeu I try to do that this evening or tomorrow. Will report back.